src/multi_vector_simulator/A0_initialization.py

```diff
@@ -498,6 +498,18 @@ def process_user_arguments(
         screen_level=screen_level,
     )
 
+    # Disable log messages of external libraries saving into the log file, unless they are ERROR or CRITICAL level
+    for ext_lib_logger in (
+        "asyncio",
+        "asyncio.coroutines",
+        "websockets.server",
+        "websockets.protocol",
+        "websockets.client",
+        "urllib3.connectionpool",
+        "PIL.PngImagePlugin",
```

Comment:
"werkzeug" for dash (it is the one of flask, on which dash piggybacks) and "pyppeteer" for the browser emulator. It is wierd to me that the commands on line https://github.com/rl-institut/multi-vector-simulator/blob/a2d27f9c5999cda3c23cf39ac542e72410248616/src/multi_vector_simulator/F2_autoreport.py#L37 does not work to suppress dash messages?